scala-generator/src/main/scala/models/play/files/ModelsJson.scala

```diff
@@ -1,94 +1,72 @@
 package scala.models.play.files
 
-import io.apibuilder.generator.v0.models.{File, InvocationForm}
+import io.apibuilder.generator.v0.models.InvocationForm
 import scala.generator.Namespaces
 
 object ModelsJson {
 
-    def uuidFormat(namespaces: Namespaces): String = s"""
-        private[${namespaces.last}] implicit val jsonReadsUUID = __.read[String].map(java.util.UUID.fromString)
+  def uuidFormat(namespaces: Namespaces): String = s"""
+    private[${namespaces.last}] implicit val jsonReadsUUID = __.read[String].map(java.util.UUID.fromString)
 
-        private[${namespaces.last}] implicit val jsonWritesUUID = new Writes[java.util.UUID] {
-            def writes(x: java.util.UUID) = JsString(x.toString)
-        }
-    """
+    private[${namespaces.last}] implicit val jsonWritesUUID = new Writes[java.util.UUID] {
+        def writes(x: java.util.UUID) = JsString(x.toString)
+    }
+  """
```

Comment:
Todo: Use the same code as in Play2Models to figure out which implicits to implement and which to use from play-json